kohese-portal/client/src/components/object-editor/format-object-editor/format-object-editor.component.spec.ts

```diff
@@ -62,7 +63,9 @@ describe('FormatObjectEditorComponent', () => {
         FormsModule,
         BrowserAnimationsModule,
         MaterialModule,
-        MarkdownModule,
+        MarkdownModule.forRoot({
+          sanitize: SecurityContext.NONE
```

Comment:
Need to evaluate the implications of this.

Reply:
Setting the SecurityContext to none was necessary for our markdown to appear correctly. Many locations were not checked, but Document Row component's title's would not show correctly without this option set.
There is, however, another sanitization process that happens in one of the pipes that markdown goes through. This security context is also set to bypass (aka ignore security issues).